sdl-core/src/test/scala/io/smartdatalake/config/ConfigLoaderTest.scala

```diff
@@ -62,6 +62,11 @@ class ConfigLoaderTest extends FlatSpec with Matchers {
     an [ConfigurationException] should be thrownBy ConfigLoader.loadConfigFromFilesystem(Seq("foo/bar"), defaultHadoopConf)
   }
 
+  it must "fail if no configuration files are found for any location provided location in the Sequence" in {
+    a [ConfigurationException] should be thrownBy ConfigLoader.loadConfigFromFilesystem(Seq(getClass.getResource("/config/config.conf").toString, getClass.getResource("/config").toURI.toString + "/error.conf"), defaultHadoopConf)
+    a [ConfigurationException] should be thrownBy ConfigLoader.loadConfigFromFilesystem(Seq(getClass.getResource("/config/config.conf").toString, getClass.getResource("/config").toURI.toString + "/empty"), defaultHadoopConf)
```

Comment:
It should also fail on an existing directory, where no configuration files are found. Can you please check?

Reply:
Added the test.

Reply:
Done.